src/batch/src/executor/insert.rs

```diff
@@ -228,6 +248,7 @@ mod tests {
             source_manager.clone(),
             Box::new(mock_executor),
             "InsertExecutor".to_string(),
+            vec![], // Ignoring insertion order
```

Comment:
I am ignoring the insertion order on purpose here, since we already validate that the insertion order works in the e2e tests. is that ok?